### ISSUE TITLE
Edge loss does not decline 

### ISSUE THREAD

**Issue by jakub-kaluza** [nv-tlabs/GSCNN#53]
I try to reproduce the results from the paper. I use the recommended hyper-parameters.

`Namespace(adam=False, amsgrad=False, arch='network.gscnn.GSCNN', att_weight=1.0, batch_weighting=False, bblur=False, best_record={'acc_cls': 0, 'val_loss': 10000000000.0, 'mean_iu': 0, 'iter': 0, 'epoch': -1, 'fwavacc': 0, 'acc': 0}, bs_mult=1, bs_mult_val=2, ckpt='/experiments/image-segmentation/GSCNN/', color_aug=0.25, crop_size=720, cv=0, dataset='cityscapes', date_str='2019_12_09_15_39_04', dual_weight=1.0, dump_augmentation_images=False, edge_weight=20.0, eval_thresholds='0.0005,0.001875,0.00375,0.005', evaluate=False, exp='GSCNN-cityspace', exp_path='/experiments/image-segmentation/GSCNN/GSCNN-cityspace/testing', gblur=True, img_wt_loss=False, joint_edgeseg_loss=True, last_record={}, local_rank=0, lr=0.01, lr_schedule='poly', maxSkip=0, max_epoch=175, momentum=0.9, ngpu=1, poly_exp=1.0, pre_size=None, repoly=1.5, rescale=1.0, restore_optimizer=False, rotate=0, scale_max=2.0, scale_min=0.5, seg_weight=1.0, sgd=True, sgd_finetuned=False, snapshot=None, start_epoch=0, syncbn=True, tb_exp_path='/experiments/image-segmentation/GSCNN/GSCNN-cityspace/testing', tb_path='/experiments/image-segmentation/GSCNN/', tb_tag='', test_mode=False, trunk='resnet101', weight_decay=0.0001, world_size=1, wt_bound=1.0)`

For training I use one Nvidia Tesla V100 Data Center GPU.

I have a following issue, all the IoUs go up, all the losses fall except for the edge loss? Why is so? I attach screenshot from my tensorboard

![tensorboardshot](https://user-images.githubusercontent.com/6080011/70793865-9deed480-1d9c-11ea-9e03-2cb9da7a6175.png)

Thanks in advance for the response.
